commitizen/formats/__init__.py

```diff
@@ -0,0 +1,94 @@
+from __future__ import annotations
+
+import sys
+
+from typing import ClassVar, Optional
+
+import importlib_metadata as metadata
+
+from commitizen.changelog import Metadata
+from commitizen.config.base_config import BaseConfig
+
+from .errors import FormatUnknown
+
+if sys.version_info >= (3, 8):
+    from typing import Protocol
+else:
+    from typing_extensions import Protocol
+
+FORMAT_ENTRYPOINT = "commitizen.changelog_format"
+DEFAULT_FORMAT = "markdown"
+DEFAULT_TEMPLATE_EXTENSION = "j2"
+
```

Comment:
Not sure whether we should put it here or in https://github.com/commitizen-tools/commitizen/blob/master/commitizen/defaults.py

Reply:
Done:
- `DEFAULT_FORMAT` moved in `commitizen.defaults.CHANGELOG_FORMAT`
- `DEFAULT_TEMPLATE_EXTENSION` just renamed into `TEMPLATE_EXTENSION` as this is not a setting and so not a default value, just here to avoid a magic value or an hardcoded extension